pydm/application.py

```diff
@@ -465,8 +457,7 @@ def get_path(self, ui_file):
         full_path = os.path.join(dirname, str(ui_file))
         return full_path
 
-    def open_relative(self, ui_file, widget, macros=None, command_line_args=[],
-                      establish_connection=True):
+    def open_relative(self, ui_file, widget, macros=None, command_line_args=[]):
```

Comment:
Want to at least catch the kwargs as to not break API?
```suggestion
    def open_relative(self, ui_file, widget, macros=None, command_line_args=[], **kwargs):
```

Reply:
Good catch!

Reply:
Maybe also warn that they're being ignored (if specified)?

Reply:
Makes sense! Thank you for looking at that. I added it at the latest commit.